src/ble/tests/TestBtpEngine.cpp

```diff
@@ -0,0 +1,179 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
```

Comment:
could you also add 
 *    Copyright (c) 2024 Project CHIP Authors
 *    Copyright (c) 2018 Google LLC.
 *    Copyright (c) 2018 Nest Labs, Inc.